packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -286,9 +297,32 @@ contract SystemConfig is OwnableUpgradeable, ISemver {
     }
 
     /// @notice Updates the unsafe block signer address. Can only be called by the owner.
+    /// @param _batcherHash New batch hash.
     /// @param _unsafeBlockSigner New unsafe block signer address.
-    function setUnsafeBlockSigner(address _unsafeBlockSigner) external onlyOwner {
+    function setSequencer(bytes32 _batcherHash, address _unsafeBlockSigner) external onlyOwner {
```

Comment:
Note that there is now only a single `setSequencer` method, rather than separate `setUnsafeBlockSigner` and `setBatcherHash`. 
This is because they are defined in the `SuperchainConfig` allowlist as key pairs, so it does not make sense to separate them out here.